internal/engine/engine_test.go

```diff
@@ -41,10 +43,8 @@ func TestCheck(t *testing.T) {
 		tcase := tcase
 		t.Run(tcase.Name, func(t *testing.T) {
 			tc := readTestCase(t, tcase.Input)
-			buf := new(bytes.Buffer)
 
-			haveOutputs, err := eng.Check(context.Background(), tc.Inputs, WithWriterTraceSink(buf))
-			t.Logf("TRACE =>\n%s", buf.String())
+			haveOutputs, err := eng.Check(context.Background(), tc.Inputs, WithZapTraceSink(zaptest.NewLogger(t)))
```

Comment:
Is there any way in which we could maybe reuse the printer used by the `compile` command to output the traces here as well? The logger output is a bit difficult to parse when trying to debug something during development.

Reply:
Good call, done!